src/cephadm/cephadm

```diff
@@ -1468,6 +1494,8 @@ def get_container(fsid, daemon_type, daemon_id, privileged=False,
         container_args.extend(monitoring_args)
     elif daemon_type == 'crash':
         ceph_args = ['-n', name]
+    elif daemon_type == 'iscsi':
+        ceph_args = []
```

Comment:
Well, unless it changed (and after the rebase I'd need to check) but anything here would be passed to the rbd-target-api.py, `-n` and `-f` might be harmless atm to the python script, but worry passing random params at it could lead to potential bugs in the future. I guess this is a plus for splitting it all out.